lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -1133,64 +1180,85 @@ def do_3d_projection(self):
                 self._facecolor3d = self._facecolors
             if self._edge_is_mapped:
                 self._edgecolor3d = self._edgecolors
+
+        needs_masking = np.any(self._invalid_vertices)
+        num_faces = len(self._faces)
+        mask = self._invalid_vertices
+
+        # Some faces might contain masked vertices, so we want to ignore any
+        # errors that those might cause
```

Comment:
Should these errors be handled within the `_proj_transform_vectors` case directly closer to where these errors would actually arise? I know there has been some work in numpy to not raise invalid/division warnings on masked elements, so is this still an issue now?

Reply:
Looks like the error does crop up in the AppVeyor build, so I'm reverting back to the original error suppression.

Reply:
Looks like the error does crop up in the AppVeyor build, so adding the original error suppression back in. I don't want to put it in `_proj_transform_vectors` since it might not always be the case that we know that we have masked out the problematic vertices outside of that function.